soc/arm/nordic_nrf/nrf52/soc.c

```diff
@@ -47,6 +47,22 @@ extern void z_arm_nmi_init(void);
 #define LOG_LEVEL CONFIG_SOC_LOG_LEVEL
 LOG_MODULE_REGISTER(soc);
 
+int soc_mcuboot_mode_set(uint8_t mode)
```

Comment:
Nothing wrong with this line of code, but the Signed-off-by: line in the commit message has the wrong format.